scripts/actions/deserialize-html.js

```diff
@@ -67,6 +68,21 @@ const headingWithCustomId = (h, node) => {
   return result;
 };
 
+const stripTranslateFrontmatter = () => {
+  const transformer = (tree) => {
+    if (tree?.children[0]?.type === 'yaml') {
+      const frontmatterObj = yaml.load(tree.children[0].value);
+      delete frontmatterObj.translate;
+      delete frontmatterObj.redirects;
```

Comment:
Do we want to remove redirects for localized pages? I would think that, if we have set up a redirect for a page in english, that we would also want that to be set up in other languges.

Reply:
this was at the recommendation of @roadlittledawn, maybe he can expand on it?

Reply:
yeah we do. we add redirects from English `.mdx` files to their localized equivalents rather than reading them directly from `src/i18n` files since they're always the same / we match them 1:1